mock_application_integration.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Moved from test-only generated mocks to ones that are exported. This is to allow interface implementors to perform better testing while mocking out some of the autopilot dependencies.